content/posts/2022-08-22-okd-on-crc.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Running OKD with CRC"
+date: 2022-08-22T09:14:10+05:30
+---
+
+It has been long time since crc release updated with OKD bundle. Last release was [1.x](https://www.okd.io/crc/#get-started) and
+not updated since then. During our 2.x development cycle, we made effort on CRC side to decouple the bundles from the release
+artifacts. It allows us to create the bundle independently for the release and upload it to separate location.
+
+Another change we did during 2.x is to add support of preset and from starting we are supporting `podman` and `openshift`
```

Comment:
we did/we made

Reply:
A "preset" is a set of configuration parameters for the bundle to be used, the assigned memory and CPU and changes behaviour if needed for the startup process.